File: java info pavansir/0copy coad/Assingment2/LeapYear.java
package Assingment2;
import java.util.Scanner;
public class LeapYear {
    public static void main(String[] args) {
        Scanner s=new Scanner(System.in);
        System.out.println("enter any year=");
        int year=s.nextInt();
        if(year%400==0||year%4==0 && year%100!=0){
            System.out.println("This year is leap year");
        }
        else{
            System.out.println("This year is not leap year");
        }
    }
}

File: coading_C_Collage_Wallah/IF-ELSE-STATEMENT.c/leapyear.c
#include<stdio.h>
int main(){

     int n;
     printf("Enter the year = ");
     scanf("%d",&n);

    if(n%4 == 0){
        printf("This  leap year is = %d,",n);
    }else{
        printf("This year is not leap = %d",n);
    }
    return 0;
}
